staging/src/k8s.io/client-go/tools/clientcmd/validation_test.go

```diff
@@ -443,22 +443,19 @@ func TestValidateAuthInfoExecWithAuthProvider(t *testing.T) {
 	test.testConfig(t)
 }
 
-func TestValidateAuthInfoExecInvalidEnv(t *testing.T) {
```

Comment:
can you keep this test case but make the invalid aspect a zero-length envvar name? a new testcase to ensure zero-length values are accepted is fine